### PR TITLE
[Doc] PR for Document LDAP support for Platform (API) (Doc-PLAT-2454)

### PR DESCRIPTION
**Pull request by lizayugabyte**

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+  ```shell
+  curl --location --request PUT 'https://10.9.140.199/api/v1/customers/f3a63f07-e3d6-4475-96e4-57a6453072e1/runtime_config/00000000-0000-0000-0000-000000000000/key/yb.security.ldap.use_ldap' \
+  --header 'X-AUTH-YW-API-TOKEN: 5182724b-1891-4cde-bcd1-b8f7a3b7331e' \
+  --header 'Content-Type: text/plain' \
+  --header 'Cookie: csrfCookie=d5cdb2b36b00fcad1f4fdb24605aee412f8dfaa0-1641544510767-641be933bf684abcade3c592' \
+  --data-raw 'true'
+  ```
+
+- Thehe LDAP port, as follows:
```
Comment (TaranSinghania-yb) [inline docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:49]: Typo(?)

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  ```shell
+  curl --location --request PUT 'https://10.9.140.199/api/v1/customers/f3a63f07-e3d6-4475-96e4-57a6453072e1/runtime_config/00000000-0000-0000-0000-000000000000/key/yb.security.ldap.ldap_port' \
+  --header 'X-AUTH-YW-API-TOKEN: 5182724b-1891-4cde-bcd1-b8f7a3b7331e' \
+  --header 'Content-Type: text/plain' \
+  --header 'Cookie: csrfCookie=d5cdb2b36b00fcad1f4fdb24605aee412f8dfaa0-1641544510767-641be933bf684abcade3c592' \
+  --data-raw '389'
+  ```
+
+- The base directory name (DN) to enable restriction of users and user groups, as follows:
```
Comment (TaranSinghania-yb) [inline docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:59]: DN stands for distinguished name

**Review by TaranSinghania-yb — COMMENTED** [review]
LGTM for pre-ui documentation. Just some minor comments.
- Would be nice to add that they can then login by specifying the common name of the user and the password to bind to the LDAP server.

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:49]: Fixed

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:59]: Changed

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md]: Fixed

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md]: Changed

**Comment by lizayugabyte** [comment]
@sridhar-yb I'm going to merge this PR. If you'd like to provide additional input, please do and I'll create a new PR for these changes.

**Pull request merged**